lib/activerecord_spanner_adapter/base.rb

```diff
@@ -60,7 +63,16 @@ def self._insert_record values
       else
         im = arel_table.compile_insert _substitute_values(values)
       end
-      connection.insert(im, "#{self} Create", primary_key || false, primary_key_value)
+      result = connection.insert(im, "#{self} Create", primary_key || false, primary_key_value)
```

Comment:
It can be both. In case of interleaved tables, the primary key will consist of multiple columns, and both the `primary_key` and `primary_key_values` will be arrays here. This line of code is used with composite primary keys when:
1. We insert a row using DML (so not mutations).
2. The table uses a composite primary key (so for example an interleaved table).

The above condition is covered by this test: https://github.com/googleapis/ruby-spanner-activerecord/blob/4a4c1d87150ae787ad024021c088c53691ae4a08/test/activerecord_spanner_interleaved_table/interleaved_tables_test.rb#L375

This line and the following 2 verify that the value that is sent to Cloud Spanner contains both the `singer_id` and `album_id` for the `Album`: https://github.com/googleapis/ruby-spanner-activerecord/blob/4a4c1d87150ae787ad024021c088c53691ae4a08/test/activerecord_spanner_interleaved_table/interleaved_tables_test.rb#L395